R-packages/covidcast/DESCRIPTION

```diff
@@ -46,8 +46,10 @@ Imports:
   maptools,
   purrr,
   rlang,
+  sf,
   tidyr,
   usmap
+
```

Comment:
@sarah-colq I'd be surprised if this was the issue, but is it this newline?

Seems like the error is may be due to[ improperly specified packages](https://github.com/r-lib/devtools/issues/1262#issuecomment-232352601), and I wonder if it's looking at the newline.

Reply:
Looks like that fixed it!